pyproject.toml

```diff
@@ -5,7 +5,7 @@ description = "Calcula la ruta más óptima para el repostaje de gasolina en un
 authors = ["jose <josevilchez4@correo.ugr.es>"]
 
 [tool.poetry.dependencies]
-python = ">=3.7,<4.0.0"
+python = "^3.9"
```

Comment:
Por favor, Jose, céntrate un poco. Para empezar, no puedes poner en un commit cosas obvias como el nombre del fichero que estás cambiando. Di por qué lo haces. Pero, lo más importante, ¡estás cambiando la versión de Python y la forma de expresarla en vez de simplemente borrar versiones inexistentes! ¿Por qué lo haces?
Por favor, tratad de entender lo que hacéis y por qué lo hacéis, y qué implicación tienen los cambios y la forma de documentarlos con mensajes de commit.

Reply:
Cierto simplemente tendría que haber borrado la versión inexistente. Lo cambio